frontend/src/helpers/string/string.helper.ts

```diff
@@ -0,0 +1,7 @@
+import prettyBytes from 'pretty-bytes';
```

Comment:
move to`frontend/src/helpers/string/get-pretty-bytes/get-pretty-bytes.helper.ts`